sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/workflow/WorkflowImpl.scala

```diff
@@ -196,7 +196,7 @@ final class WorkflowImpl(system: ActorSystem, val services: Map[String, Workflow
     init.userState match {
       case Some(state) =>
         val decoded = service.messageCodec.decodeMessage(state)
-        router._internalSetInitState(decoded)
+        router._internalSetInitState(decoded, finished = false) //TODO get this from the init message
```

Comment:
If we are ok, with the solution I will update the proxy and then use the proper value it in this or some next PR.